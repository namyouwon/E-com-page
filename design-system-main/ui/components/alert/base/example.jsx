// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import Alert from '../';
import { UtilityIcon } from '../../icons/base/example';
import EmptyLink from '../../../shared/empty-link/';
import Heading from '../../../shared/heading/';

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <Alert>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="user"
        />
        <Heading level="2">
          Logged in as John Smith (johnsmith@acme.com).{' '}
          <EmptyLink>Log out</EmptyLink>
        </Heading>
      </Alert>
    )
  }
];

export let states = [
  {
    id: 'theme-info',
    label: 'Default (Theme Class  - Deprecated)',
    element: (
      <Alert isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="user"
        />
        <Heading level="2">
          Logged in as John Smith (johnsmith@acme.com).{' '}
          <EmptyLink>Log out</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'warning',
    label: 'Warning',
    element: (
      <Alert type="warning">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="warning"
        />
        <Heading level="2">
          Your browser is outdated. Your Salesforce experience may be degraded.{' '}
          <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-warning',
    label: 'Warning (Theme Class  - Deprecated)',
    element: (
      <Alert type="warning" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="warning"
        />
        <Heading level="2">
          Your browser is outdated. Your Salesforce experience may be degraded.{' '}
          <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'error',
    label: 'Error',
    element: (
      <Alert type="error">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="error"
        />
        <Heading level="2">
          Your browser is currently not supported. Your Salesforce may be
          degraded. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-error',
    label: 'Error (Theme Class  - Deprecated)',
    element: (
      <Alert type="error" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="error"
        />
        <Heading level="2">
          Your browser is currently not supported. Your Salesforce may be
          degraded. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'offline',
    label: 'Offline',
    element: (
      <Alert type="offline">
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="offline"
        />
        <Heading level="2">
          You are in offline mode. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  },
  {
    id: 'theme-offline',
    label: 'Offline (Theme Class  - Deprecated)',
    element: (
      <Alert type="offline" isLegacy>
        <UtilityIcon
          containerClassName="slds-m-right_x-small"
          className="slds-icon_x-small"
          assistiveText={false}
          symbol="offline"
        />
        <Heading level="2">
          You are in offline mode. <EmptyLink>More Information</EmptyLink>
        </Heading>
      </Alert>
    )
  }
];
