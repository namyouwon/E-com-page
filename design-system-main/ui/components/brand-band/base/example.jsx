// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';

import BrandBand from '../';

export default <BrandBand size="medium" />;

export let examples = [
  {
    id: 'brand-band-small',
    label: 'Small',
    element: <BrandBand size="small" />
  },
  {
    id: 'brand-band-large',
    label: 'Large',
    element: <BrandBand size="large" />
  },
  {
    id: 'brand-band-no-image',
    label: 'No Image',
    element: <BrandBand size="none" />
  },
  {
    id: 'brand-band-large-group',
    label: 'Large Group',
    element: <BrandBand size="large" image="group" />
  },
  {
    id: 'brand-band-large-user',
    label: 'Large User',
    element: <BrandBand size="large" image="user" />
  }
];
