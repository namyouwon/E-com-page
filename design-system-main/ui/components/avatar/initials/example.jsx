// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import { Avatar } from '../base/example';
import { AvatarInitials } from '../';

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <Avatar>
        <AvatarInitials
          iconClassName="slds-icon-standard-account"
          title="company name"
          initials="Ac"
        />
      </Avatar>
    )
  }
];

export let examples = [
  {
    id: 'linked-avatar-initials',
    label: 'Wrapped in a link',
    element: (
      <a href="#" onClick={e => e.preventDefault()}>
        <Avatar>
          <AvatarInitials
            iconClassName="slds-icon-standard-account"
            title="company name"
            initials="Ac"
          />
        </Avatar>
      </a>
    )
  }
];
