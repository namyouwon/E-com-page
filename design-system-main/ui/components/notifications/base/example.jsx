// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license

import React from 'react';
import uniqueId from 'lodash.uniqueid';
import ButtonIcon from '../../button-icons/';
import { StandardIcon } from '../../icons/standard/example';
import classNames from 'classnames';

/// ///////////////////////////////////////////
// State Constructor(s)
/// ///////////////////////////////////////////

let NotificationContainer = props => (
  <div className="slds-notification-container">{props.children}</div>
);

let Notification = props => {
  const bodyUniqueId = uniqueId('dialog-body-id-');
  const headingId = uniqueId('noti-')

  return (
    <section
      className="slds-notification"
      role="dialog"
      aria-labelledby={headingId}
      aria-describedby={bodyUniqueId}
    >
      <div className="slds-notification__body" id={bodyUniqueId}>
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={e => e.preventDefault()}
        >
          <StandardIcon
            containerClassName="slds-media__figure"
            className="slds-icon_small"
            symbol={props.type}
            title={props.type}
          />
          <div className="slds-media__body">
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id={headingId}
            >
              <span className="slds-assistive-text">
                {`${props.type} notification:`}
              </span>
              {props.title}
            </h2>
            <p>{props.description}</p>
          </div>
        </a>
        <ButtonIcon
          className="slds-button_icon-container slds-notification__close"
          symbol="close"
          assistiveText={`Dismiss ${props.title} notification`}
          title={`Dismiss ${props.title} notification`}
        />
      </div>
      {props.footer && (
        <footer
          className={classNames(
            'slds-notification__footer',
            props.footerClassName
          )}
        >
          {props.footer}
        </footer>
      )}
    </section>
  );
};

/// ///////////////////////////////////////////
// Export
/// ///////////////////////////////////////////

export default [
  {
    id: 'default',
    label: 'Default',
    element: (
      <div className="demo-only demo-only_viewport" style={{ height: '4.5rem' }}>
        <NotificationContainer>
          <div
            aria-live="assertive"
            aria-atomic="true"
            className="slds-assistive-text"
          >
            event notification: Tesla - Renewal meeting
          </div>
          <Notification
            type="event"
            title="Tesla - Renewal meeting"
            description="Event at 11:00am on Jan 8"
          />
        </NotificationContainer>
      </div>
    )
  }
];

export let examples = [
  {
    id: 'task-notification',
    label: 'Task Notification',
    element: (
      <div
        className="demo-only demo-only_viewport"
        style={{ height: '4.5rem' }}
      >
        <NotificationContainer>
          <div
            aria-live="assertive"
            aria-atomic="true"
            className="slds-assistive-text"
          >
            task notification: Call: Jane Johnson
          </div>
          <Notification
            type="task"
            title="Call: Jane Johnson"
            description="Task due on Jan 8"
          />
        </NotificationContainer>
      </div>
    )
  },
  {
    id: 'stacked-2',
    label: 'Stacked Notifications',
    element: (
      <div className="demo-only demo-only_viewport" style={{ height: '15rem' }}>
        <NotificationContainer>
          <div
            aria-live="assertive"
            aria-atomic="true"
            className="slds-assistive-text"
          >
            task notification: Call One: Jane Johnson
          </div>
          <Notification
            type="task"
            title="Call One: Jane Johnson"
            description="Task due on Jan 8"
          />
          <Notification
            type="event"
            title="Call Two: Tesla - Renewal meeting"
            description="Event at 11:00am on Jan 8"
          />
        </NotificationContainer>
      </div>
    )
  },
  {
    id: 'stacked-3',
    label: 'Three Stacked Notifications',
    element: (
      <div className="demo-only demo-only_viewport" style={{ height: '15rem' }}>
        <NotificationContainer>
          <div
            aria-live="assertive"
            aria-atomic="true"
            className="slds-assistive-text"
          >
            task notification: Call Two: Jane Johnson
          </div>
          <Notification
            type="task"
            title="Call Two: Jane Johnson"
            description="Task due on Jan 8"
          />
          <Notification
            type="event"
            title="Call Three: Tesla - Renewal meeting"
            description="Event at 11:00am on Jan 8"
          />
          <Notification
            type="task"
            title="Call Four: Jim Jameson"
            description="Task due on Jan 8"
          />
        </NotificationContainer>
      </div>
    )
  },
  {
    id: 'overflow-six',
    label: 'Six Stacked Notifications',
    element: (
      <div className="demo-only demo-only_viewport" style={{ height: '17rem' }}>
        <NotificationContainer>
          <div
            aria-live="assertive"
            aria-atomic="true"
            className="slds-assistive-text"
          >
            task notification: Call Five: Jane Johnson
          </div>
          <Notification
            type="task"
            title="Call Five: Jane Johnson"
            description="Task due on Jan 8"
          />
          <Notification
            type="event"
            title="Call Six: Tesla - Renewal meeting"
            description="Event at 11:00am on Jan 8"
          />
          <Notification
            type="task"
            title="Call Seven: Jim Jameson"
            description="Task due on Jan 8"
          />
          <Notification
            type="task"
            title="Call Eight: Jane Johnson"
            description="Task due on Jan 8"
          />
          <Notification
            type="event"
            title="Call Nine: Tesla - Renewal meeting"
            description="Event at 11:00am on Jan 8"
          />
          <Notification
            type="task"
            title="Call Ten: Jim Jameson"
            description="Task due on Jan 8"
          />
        </NotificationContainer>
      </div>
    )
  }
];
