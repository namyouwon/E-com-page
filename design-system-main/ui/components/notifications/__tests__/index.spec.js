// Copyright (c) 2015-present, salesforce.com, inc. All rights reserved
// Licensed under BSD 3-Clause - see LICENSE.txt or git.io/sfdc-license
/* eslint-env jest */

// import React from 'react';
import createHelpers from '../../../../jest.helpers';
import { getDisplayElementById } from '../../../shared/helpers';
import { examples } from '../base/example';

const { matchesMarkup } = createHelpers(__dirname);

describe('Notifications', () => {
  it('renders a single notification', () =>
    matchesMarkup(getDisplayElementById(examples, 'task-notification'))
  )

  it('renders two stacked notifications', () =>
    matchesMarkup(getDisplayElementById(examples, 'stacked-2'))
  )

  it('renders three stacked notifications', () =>
    matchesMarkup(getDisplayElementById(examples, 'stacked-3'))
  )

  it('renders six stacked notifications', () =>
    matchesMarkup(getDisplayElementById(examples, 'overflow-six'))
  )
});
