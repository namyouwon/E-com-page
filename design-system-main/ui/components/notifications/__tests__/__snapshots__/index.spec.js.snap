// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notifications renders a single notification 1`] = `
<div
  className="demo-only demo-only_viewport"
  style={
    Object {
      "height": "4.5rem",
    }
  }
>
  <div
    className="slds-notification-container"
  >
    <div
      aria-atomic="true"
      aria-live="assertive"
      className="slds-assistive-text"
    >
      task notification: Call: Jane Johnson
    </div>
    <section
      aria-describedby="dialog-body-id-1"
      aria-labelledby="noti-2"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-1"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-2"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call: Jane Johnson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call: Jane Johnson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call: Jane Johnson notification
          </span>
        </button>
      </div>
    </section>
  </div>
</div>
`;

exports[`Notifications renders six stacked notifications 1`] = `
<div
  className="demo-only demo-only_viewport"
  style={
    Object {
      "height": "17rem",
    }
  }
>
  <div
    className="slds-notification-container"
  >
    <div
      aria-atomic="true"
      aria-live="assertive"
      className="slds-assistive-text"
    >
      task notification: Call Five: Jane Johnson
    </div>
    <section
      aria-describedby="dialog-body-id-13"
      aria-labelledby="noti-14"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-13"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-14"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call Five: Jane Johnson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Five: Jane Johnson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Five: Jane Johnson notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-15"
      aria-labelledby="noti-16"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-15"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-event slds-media__figure"
            title="event"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#event"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-16"
            >
              <span
                className="slds-assistive-text"
              >
                event notification:
              </span>
              Call Six: Tesla - Renewal meeting
            </h2>
            <p>
              Event at 11:00am on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Six: Tesla - Renewal meeting notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Six: Tesla - Renewal meeting notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-17"
      aria-labelledby="noti-18"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-17"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-18"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call Seven: Jim Jameson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Seven: Jim Jameson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Seven: Jim Jameson notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-19"
      aria-labelledby="noti-20"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-19"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-20"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call Eight: Jane Johnson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Eight: Jane Johnson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Eight: Jane Johnson notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-21"
      aria-labelledby="noti-22"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-21"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-event slds-media__figure"
            title="event"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#event"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-22"
            >
              <span
                className="slds-assistive-text"
              >
                event notification:
              </span>
              Call Nine: Tesla - Renewal meeting
            </h2>
            <p>
              Event at 11:00am on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Nine: Tesla - Renewal meeting notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Nine: Tesla - Renewal meeting notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-23"
      aria-labelledby="noti-24"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-23"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-24"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call Ten: Jim Jameson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Ten: Jim Jameson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Ten: Jim Jameson notification
          </span>
        </button>
      </div>
    </section>
  </div>
</div>
`;

exports[`Notifications renders three stacked notifications 1`] = `
<div
  className="demo-only demo-only_viewport"
  style={
    Object {
      "height": "15rem",
    }
  }
>
  <div
    className="slds-notification-container"
  >
    <div
      aria-atomic="true"
      aria-live="assertive"
      className="slds-assistive-text"
    >
      task notification: Call Two: Jane Johnson
    </div>
    <section
      aria-describedby="dialog-body-id-7"
      aria-labelledby="noti-8"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-7"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-8"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call Two: Jane Johnson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Two: Jane Johnson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Two: Jane Johnson notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-9"
      aria-labelledby="noti-10"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-9"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-event slds-media__figure"
            title="event"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#event"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-10"
            >
              <span
                className="slds-assistive-text"
              >
                event notification:
              </span>
              Call Three: Tesla - Renewal meeting
            </h2>
            <p>
              Event at 11:00am on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Three: Tesla - Renewal meeting notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Three: Tesla - Renewal meeting notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-11"
      aria-labelledby="noti-12"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-11"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-12"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call Four: Jim Jameson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Four: Jim Jameson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Four: Jim Jameson notification
          </span>
        </button>
      </div>
    </section>
  </div>
</div>
`;

exports[`Notifications renders two stacked notifications 1`] = `
<div
  className="demo-only demo-only_viewport"
  style={
    Object {
      "height": "15rem",
    }
  }
>
  <div
    className="slds-notification-container"
  >
    <div
      aria-atomic="true"
      aria-live="assertive"
      className="slds-assistive-text"
    >
      task notification: Call One: Jane Johnson
    </div>
    <section
      aria-describedby="dialog-body-id-3"
      aria-labelledby="noti-4"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-3"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-task slds-media__figure"
            title="task"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#task"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-4"
            >
              <span
                className="slds-assistive-text"
              >
                task notification:
              </span>
              Call One: Jane Johnson
            </h2>
            <p>
              Task due on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call One: Jane Johnson notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call One: Jane Johnson notification
          </span>
        </button>
      </div>
    </section>
    <section
      aria-describedby="dialog-body-id-5"
      aria-labelledby="noti-6"
      className="slds-notification"
      role="dialog"
    >
      <div
        className="slds-notification__body"
        id="dialog-body-id-5"
      >
        <a
          className="slds-notification__target slds-media"
          href="#"
          onClick={[Function]}
        >
          <span
            className="slds-icon_container slds-icon-standard-event slds-media__figure"
            title="event"
          >
            <svg
              aria-hidden={true}
              className="slds-icon slds-icon_small"
            >
              <use
                xlinkHref="/assets/icons/standard-sprite/svg/symbols.svg#event"
              />
            </svg>
            <span
              className="slds-assistive-text"
            >
              Description of icon when needed
            </span>
          </span>
          <div
            className="slds-media__body"
          >
            <h2
              className="slds-text-heading_small slds-m-bottom_xx-small"
              id="noti-6"
            >
              <span
                className="slds-assistive-text"
              >
                event notification:
              </span>
              Call Two: Tesla - Renewal meeting
            </h2>
            <p>
              Event at 11:00am on Jan 8
            </p>
          </div>
        </a>
        <button
          className="slds-button slds-button_icon slds-button_icon-container slds-notification__close"
          onClick={[Function]}
          title="Dismiss Call Two: Tesla - Renewal meeting notification"
        >
          <svg
            aria-hidden={true}
            className="slds-button__icon"
          >
            <use
              xlinkHref="/assets/icons/utility-sprite/svg/symbols.svg#close"
            />
          </svg>
          <span
            className="slds-assistive-text"
          >
            Dismiss Call Two: Tesla - Renewal meeting notification
          </span>
        </button>
      </div>
    </section>
  </div>
</div>
`;
